docs/source/user-guide/tasks/use-conda-with-travis-ci.rst

```diff
@@ -123,3 +122,34 @@ Travis CI with conda.
 
 For an example project building conda packages on AppVeyor, see
 https://github.com/rmcgibbo/python-appveyor-conda-example.
+
+Bootstrap your environment
+==========================
+
+If you encounter the "RemoveError" error where ``conda update conda`` tries to
```

Comment:
Let's remove the whole "RemoveError" thing.  This approach is just plain better and should be the recommended way.